features/admin.connections.v1/components/authenticators/push/quick-start.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2022-2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { useRequiredScopes } from "@wso2is/access-control";
+import ApplicationSelectionModal from "@wso2is/admin.applications.v1/components/application-selection-modal";
+import {
+    VerticalStepper,
+    VerticalStepperStepInterface
+} from "@wso2is/admin.core.v1/components/vertical-stepper/vertical-stepper";
+import { FeatureConfigInterface } from "@wso2is/admin.core.v1/models";
+import { AppState } from "@wso2is/admin.core.v1/store";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { GenericIcon, Heading, Link, PageHeader, Text } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Grid } from "semantic-ui-react";
+import BuildLoginFlowIllustration from "./assets/build-login-flow.png";
+
+/**
+ * Prop types of the component.
+ */
+type PushAuthQuickStartPropsInterface = IdentifiableComponentInterface;
+
+/**
+ * Quick start content for the Push authenticator.
+ *
+ * @param props - Props injected into the component.
+ *
+ * @returns Push Authenticator Quick start component.
+ */
+const PushAuthQuickStart: FunctionComponent<PushAuthQuickStartPropsInterface> = (
+    props: PushAuthQuickStartPropsInterface
+): ReactElement => {
+
+    const {
+        [ "data-componentid" ]: componentId = "push-authenticator-quick-start"
```

Comment:
Shall we move this to default props?

Reply:
Default props are now deprecated and will be removed in the future react release. Therefore it is recommended to use JavaScript default parameters instead.

We are already following this pattern.

[1] https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md
[2] https://github.com/wso2/identity-apps/blob/66cb2e19b5ba814e6e3dd9cad8fd094728ed237f/features/admin.application-templates.v1/components/application-creation-adapter.tsx#L59-L64